docs/reference/ImageDraw.rst

```diff
@@ -254,6 +254,22 @@ Methods
     :param outline: Color to use for the outline.
     :param fill: Color to use for the fill.
 
+
+.. py:method:: ImageDraw.regular_polygon(*, b_box, nb_sides, rotation=0, fill=None, outline=None)
```

Comment:
Nits:

Let's call it `bbox` throughout, that's what the rest of the codebase uses.

Also, `n_things` is used quite a lot, and there's very little `nb_things`:

 ```suggestion
.. py:method:: ImageDraw.regular_polygon(*, bbox, n_sides, rotation=0, fill=None, outline=None)
```

Reply:
I would go with `xy` here, as while most of the code base uses `bbox`, the ImageDraw functions use `xy`: https://pillow.readthedocs.io/en/stable/reference/ImageDraw.html#PIL.ImageDraw.ImageDraw.rectangle

```suggestion
.. py:method:: ImageDraw.regular_polygon(*, xy, n_sides, rotation=0, fill=None, outline=None)
```